sql/schema.go

```diff
@@ -0,0 +1,202 @@
+package sql
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	godiffpatch "github.com/sourcegraph/go-diff-patch"
+	"go.uber.org/zap"
+)
+
+const (
+	SchemaPath        = "schema/schema.sql"
+	UpdatedSchemaPath = "schema/schema.sql.updated"
+)
+
+// LoadSchema loads the schema embedded in the executable.
+func LoadSchema(fsys fs.FS, migrations []Migration) (*Schema, error) {
+	text, err := fs.ReadFile(fsys, SchemaPath)
+	if err != nil {
+		return nil, fmt.Errorf("error reading schema file %s: %w", SchemaPath, err)
+	}
+	return &Schema{Script: string(text), Migrations: migrations}, nil
+}
+
+// LoadDBSchemaScript retrieves the database schema as text.
+func LoadDBSchemaScript(db Executor, ignoreRx string) (string, error) {
+	var (
+		err   error
+		ignRx *regexp.Regexp
+		sb    strings.Builder
+	)
+	if ignoreRx != "" {
+		ignRx, err = regexp.Compile(ignoreRx)
+		if err != nil {
+			return "", fmt.Errorf("error compiling table ignore regexp %q: %w",
+				ignoreRx, err)
+		}
+	}
+	version, err := version(db)
+	if err != nil {
+		return "", err
+	}
+	fmt.Fprintf(&sb, "PRAGMA user_version = %d;\n", version)
+	if _, err = db.Exec(
+		// Type is either 'index' or 'table', we want tables
+		// to go first. Also, we ignore _litestream tables
```

Comment:
I think the comment is old. The `_litestream` tables can be filtered out with regex now, no?
```suggestion
		// to go first.
```

Reply:
Thanks, will fix

Reply:
Fixed